packages/cli/test/integration/generators/controller.integration.js

```diff
@@ -117,21 +118,61 @@ describe('lb4 controller', () => {
           ),
           dir => {
             tmpDir = dir;
-            testUtils.givenAnApplicationDir(tmpDir);
-            fs.writeFileSync(
-              testUtils.givenAModelPath(tmpDir, 'foo.model.ts'),
-              '--DUMMY VALUE--',
-            );
-            fs.writeFileSync(
-              testUtils.givenARepositoryPath(tmpDir, 'bar.repository.ts'),
-              '--DUMMY VALUE--',
-            );
+            givenModelAndRepository(tmpDir);
           },
         )
         .then(() => {
           return checkRestCrudContents(tmpDir);
         });
     });
+    describe('modelNamePlural', () => {
+      it('is inferred to be plural form of model name', () => {
+        let tmpDir;
+        return testUtils
+          .runGeneratorWith(
+            generator,
+            Object.assign(
+              {
+                modelName: 'Foo',
```

Comment:
Please use a multi-word name to verify how it is converted to an URL. It's also better to use real-world names instead of generic `Foo`.

A proposed model name: `ProductReview`.